example/model.py

```diff
@@ -1,141 +0,0 @@
-import numpy as np
```

Comment:
We decided to delete this file (2 weeks ago), just confirm

Reply:
The model.py is not used.
np is unused within the file. I suppose it could go.